tests/cli/workflow/taxprofiler/test_cli_taxprofiler_compound_commands.py

```diff
@@ -49,3 +49,30 @@ def test_taxprofiler_start(
 
     # THEN command should not include resume flag
     assert "-resume" not in caplog.text
+
+
+def test_taxprofiler_start_available(
```

Comment:
I would also add a test when the CGError is raised

Reply:
@sofstam I mean this kind of test, see Sebastin's comment

Reply:
Ok, I see.

Reply:
We discussed with @sofstam and decided that the distinction between `CgError` and a general `Exception` is not needed. We decided to catch only `Exceptions`, so this test will no longer be required